source/server/listener_impl.cc

```diff
@@ -537,6 +538,13 @@ void ListenerImpl::buildUdpListenerFactory(Network::Socket::Type socket_type,
   }
 
   udp_listener_config_ = std::make_shared<UdpListenerConfigImpl>(config_.udp_listener_config());
+  ProtobufTypes::MessagePtr udp_packet_packet_writer_config;
+  if (config_.udp_listener_config().has_udp_packet_packet_writer_config()) {
+    auto* factory_factory = Config::Utility::getFactory<Network::UdpPacketWriterFactoryFactory>(
```

Comment:
factory_factory :-/

Given all the confusion you and I had, do you think we should find somewhere to explain the extra level of indirection?

Reply:
Good point. I updated the FactoryFactory class comment in envoy/network/udp_packet_writer_handler.h